### ISSUE TITLE
Setup startup config

### ISSUE THREAD

**Issue by sigrlami** [smaccoun/elm-haskell-scaffold#10]
I see we're relying on environment arguments when starting an application.

It's much easier to automate with argument keys to the app or even simple YAML config for different options, so you can run different development modes and configs in agile way and have versioning for troubleshooting.

**Comment by smaccoun** [comment]
@sigrlami Yeah exactly, was hoping you could help with setting that part up. Just made a comment on #1 . I don't have a clear vision yet on exactly what all the startup/make scripts should be, but this is the first thing I was thinking. Can either collapse this issue to #1 or possibly we will have a few scripts and can refine exactly what is over time

**Comment by sigrlami** [comment]
@smaccoun scripts for build/deploy all, config for actual app development. You'll need AWS credentials, Auth0 and maybe others, better to keep them in config.

**Comment by jetaggart** [comment]
Is there a `dotenv` equivalent or something we can build easily? https://www.npmjs.com/package/dotenv

**Comment by sigrlami** [comment]
@jetaggart Yesod has this thing built-in but never seen a separate package. I think not so difficult to implement, maybe copy some things straight from Yesod.

**Comment by sigrlami** [comment]
Example:
```

**Comment by smaccoun** [comment]
I added config to webpack in Elm and inject as flags. Would probably be nice to have one yaml that directs config for front-end and back-end if we can figure that out
